src/main/resources/util/thresholds.properties

```diff
@@ -21,12 +14,21 @@ description.unstableNew=If the number of new warnings exceeds a threshold then a
                    considered as unstable.
 description.failedNew=If the number of new warnings exceeds a threshold then a build is \
                    considered as failed.
-description.onlyUseStableBuildsAsReference=The number of new warnings will be calculated based on the last \
-                   stable build, allowing reverts of unstable builds where the number of warnings was decreased. \
-                   Note that the last stable build is evaluated only by inspecting the unit test failures. The \
-                   static analysis results are not considered.
-title.usePreviousBuildAsReference=Use previous build as reference
-description.usePreviousBuildAsReference=If set then the number of new warnings will always be calculated based on the \
-            previous build. Otherwise the reference build (i.e., the build that introduced the new warnings) is used \
-            to calculate the new warnings.
-compute.new.title=Compute new warnings (based on reference build)
+compute.new.title=Compute new warnings (based on a reference build, see below; by default this is the last successful \
+                   build)
```

Comment:
Also: this is a label and should be as short as possible, maybe we should add an additional help/description that makes the naming of a reference build more clear.